tests/fixtures/jats/03665-v1-vor.xml

```diff
@@ -0,0 +1 @@
e chosen?</p><p>4) <xref ref-type="fig" rid="fig1">Figure 1</xref>. Why are the tracks shorter and sparser for the smallest particles, and also for complex-1 relative to beta-galactosidase and ribosomes? One might expect the smaller particles to move more. Does the plot show only the second stage of correction for gamma-secretase? Some comment on this would help, even if these are just arbitrary selections from the data sets.</p><p>5) It would be good to have an idea of how many particles were grouped for tracking in each case.</p><p>6) <xref ref-type="fig" rid="fig2">Figure 2</xref>. The striped plots of relative weight are difficult to understand. Why are they shown as bands of variable width? Does this indicate some range of values the extent of which varies with frequency?</p><p>7) <xref ref-type="fig" rid="fig2s1">Figure 2–figure supplement 1</xref>. How can the Guinier plot increase with frequency (image 1 of beta-galactosidase)?</p><p>8) Is it possible to estimate the extent to which the resolution of smaller particles is limited by out-of-plane rotations, which are not corrected?</p></body></sub-article><sub-article article-type="reply" id="SA2"><front-stub><article-id pub-id-type="doi">10.7554/eLife.03665.009</article-id><title-group><article-title>Author response</article-title></title-group></front-stub><body><p><italic>1) The description of how linear tracks are fitted through particles in each frame is vague. As this is a method-oriented paper, a detailed description is necessary</italic>.</p><p>The following paragraph was added:</p><p>“More specifically, for each particle <italic>p</italic>, the program independently minimizes the least squares target below for both the X- and Y components of the particle coordinates in the field of view:</p><p><inline-formula><mml:math id="inf1"><mml:mrow><mml:munder><mml:mrow><mml:mi>m</mml:mi><mml:mi>i</mml:mi><mml:mi>n</mml:mi></mml:mrow><mml:mrow><mml:msub><mml:mi>α</mml:mi><mml:mi>p</mml:mi></mml:msub><mml:mo>,</mml:mo><mml:msub><mml:mi>β</mml:mi><mml:mi>p</mml:mi></mml:msub></mml:mrow></mml:munder><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mstyle displaystyle="true"><mml:munder><mml:mo>∑</mml:mo><mml:mrow><mml:mi>p</mml:mi><mml:mo>'</mml:mo></mml:mrow></mml:munder><mml:mrow><mml:msub><mml:mi>w</mml:mi><mml:mrow><mml:mi>p</mml:mi><mml:mo>'</mml:mo></mml:mrow></mml:msub><mml:mstyle displaystyle="true"><mml:munder><mml:mo>∑</mml:mo><mml:mi>f</mml:mi></mml:munder><mml:mrow><mml:msup><mml:mrow><mml:mrow><mml:mo>(</mml:mo><mml:mrow><mml:mi>Δ</mml:mi><mml:msub><mml:mi>x</mml:mi><mml:mrow><mml:mi>p</mml:mi><mml:mo>'</mml:mo></mml:mrow></mml:msub><mml:mo>−</mml:mo><mml:msub><mml:mi>α</mml:mi><mml:mi>p</mml:mi></mml:msub><mml:mo>−</mml:mo><mml:msub><mml:mi>β</mml:mi><mml:mi>p</mml:mi></mml:msub><mml:mi>f</mml:mi></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:mrow><mml:mn>2</mml:mn></mml:msup></mml:mrow></mml:mstyle></mml:mrow></mml:mstyle></mml:mrow><mml:mo>)</mml:mo></mml:mrow></mml:mrow></mml:math></i
```

Comment:
This is the old XML. You need to drop in the new XML: http://s3.amazonaws.com/elife-articles-renamed/samples05/article-xml/elife-03665-v1.xml